src/styles/transitions.js

```diff
@@ -28,30 +33,39 @@ export const durations = {
   leavingScreen: 195,
 };
 
+/**
+ * @param {string|Array} props
+ * @param {object} param
+ * @param {string} param.prop
+ * @param {number} param.duration
+ * @param {string} param.easing
+ * @param {number} param.delay
+*/
 export default {
-  multi(property, duration, delay, easeFunction) {
-    easeFunction = easeFunction || easing.easeInOut;
-
-    if (property && Array.isArray(property)) {
-      let transitions = '';
-      for (let i = 0; i < property.length; i += 1) {
-        if (transitions) transitions += ',';
-        transitions += this.create(property[i], duration, delay, easeFunction);
-      }
-
-      return transitions;
-    }
+  easing: easingInternal,
 
-    return this.create(duration, property, delay, easeFunction);
-  },
+  duration: durationIntenal,
 
-  create(property, duration, delay, easeFunction) {
-    duration = duration || '300ms';
-    property = property || 'all';
-    delay = delay || '0ms';
-    easeFunction = easeFunction || easing.easeInOut;
+  create(props = ['all'], {
+    duration = durationIntenal.standard,
+    easing = easingInternal.easeInOut,
+    delay = 0,
+    ...other
+  } = {}) {
+    invariant(typeof props === 'string' || Array.isArray(props),
```

Comment:
Yes, I think that we want. My only question is. Why invariant and not warning? To put it differently. Do we want to throw in the production environnement?